src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -70,7 +70,23 @@
  *   &lt;property name=&quot;tokens&quot; value=&quot;COMMA, SEMI&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ *<pre>
+ * public class MyTest {
+ *  public static void myTest() {
+ *      if (foo) { // OK, 'if' followed by whitespace
```

Comment:
Remove text after "ok" or " violation" in this and other lines. On exact code there is no "if".

Reply:
Fixed.